sap/cli/atc.py

```diff
@@ -97,6 +138,25 @@ def run(connection, args):
     except KeyError:
         raise SAPCliError(f'Unknown type: {args.type}')
 
+    printer_format_mapping = {
+        'txt': print_worklist_to_stream,
+        'html': print_worklist_as_html_to_stream,
+        'xml': print_worklist_as_checkstyle_xml_to_stream
+    }
+    try:
+        printer = printer_format_mapping[args.format]
+    except KeyError:
+        raise SAPCliError(f'Unknown format: {args.format}')
+
+    severity_mapping = None
+    if args.format == 'xml':
+        severity_mapping = args.severity_mapping or os.environ.get('SEVERITY_MAPPING')
```

Comment:
I could not find the environment variable SEVERITY_MAPPING in the documentation.